future_challenges_for_RSEng.tex

```diff
@@ -350,26 +351,57 @@ \subsection{Development of the RSE community}
 load and frustration stemming from the difficulty to prompt the system effectively
 and from debugging the generated code~\cite{Simkute2025}.
+Many factors can influence AI tools adoption~\cite{Russo2024}.
 The potential of AI to be a democratising technology can only be realised
 when the algorithmic divide along with the digital divide is addressed~\cite{Yu2020}.
-
-Another risk of the widespread use of AI tools is that community resources
-such as \eg\ Stack Overflow and Wikipedia become poorer because there are fewer incentives to maintain them.
+We also note members of the SE community have called for improving LLM literacy
+by adapting SE curricula~\cite{Kirova2024}.
+
+Besides the direct impact in research software and further research output,
+we see a number of risks.
+The workshop participants suggested that the emergence of AI and automation might reduce critical thinking and imagination.
+We address this in \autoref{sec:complexity}.
+Another risk is that the widespread use of AI tools could make community resources
+such as \eg\ Stack Overflow and Wikipedia (which are used to train AI models) poorer
+due to fewer incentives to maintain them, and could compromise LLMs via `model collapse'
+when AI-generated content is added to these websites and becomes part of the training data~\cite{Shumailov2024}.
+Further risks that were later discussed include an intensified replication crisis, a de-democratization of research, and loss of trust in science.
+Due to the non-deterministic nature of generative AI, we expect AI-assisted simulations and and data analysis to become less reproducible.
+This is not an entirely new issue, and similar concerns exist in the HPC community,
+where reproducing simulation data and data analysis workflows presents tough challenges~\cite{Antunes2024}.
+Professional RSEs could safeguard reproducibility by designing, evaluating, and monitoring appropriate workflows.
+When applicable, standardized and version-controlled models should be preferred
+over self-developed models, and self-developed models should be treated like any
+other research output (i.e. publish the trained model, training data and training workflow).
+Future research will require significant computing and storage resources, as well as useful data and models,
+therefore it will be more driven by institutions able to provide these.
+Finally, an increasing risk is the loss of trust in science,
+in particular due to the automation of the peer-reviewing process~\cite{Naddaf2025}
+and the malicious use of hidden prompts~\cite{Gibney2025}.
+As researchers increase their reliance on AI assistants,
+RSEs will be freed from repeatedly crafting similar individual elements,
+but will be spending more of their time in designing and reviewing software written by other humans and AI tools.
+To address these risks, the RSE of the future could be not only closely overlooking every part
```

Comment:
```suggestion
To address these risks, the RSE of the future could be not only closely overseeing every part
```

I think this means overseeing (keeping an eye on / looking after) as opposed to overlooking (missing)?